HotSpot/View Controllers/SignUpViewController.h

```diff
@@ -0,0 +1,22 @@
+//
+//  SignUpViewController.h
+//  HotSpot
+//
+//  Created by aaronm17 on 7/16/19.
+//  Copyright © 2019 aodemuyi. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SignUpViewController : UIViewController
+
+@property (weak, nonatomic) IBOutlet UITextField *username;
+@property (weak, nonatomic) IBOutlet UITextField *password;
+@property (weak, nonatomic) IBOutlet UITextField *fullName;
+@property (weak, nonatomic) IBOutlet UITextField *licensePlate;
```

Comment:
why licensePlate? just curious...
we would also need a phone number since you want to implement texting. In addition, you can use the phone number to authorize the user (by sending a text message with a code and having them verify it)

Reply:
the "authorizing the user by sending a code" would require the use of an API, like Twilio. That shouldn't block pushing this code to master, so you can make the other changes first and work on this later in the week